src/test/java/com/learning/pages/SwagLabsLoginPage.java

```diff
@@ -0,0 +1,37 @@
+package com.learning.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class SwagLabsLoginPage {
+
+  public static final String LOGIN_PAGE_URL = "https://www.saucedemo.com/index.html";
+
+  @FindBy(id = "user-name")
+  private WebElement usernameTextBox;
+
+  @FindBy(id = "password")
+  private WebElement passwordTextBox;
+
+  @FindBy(tagName = "h3")
```

Comment:
`h3` jest zbyt ogólnym selektorem, sugeruję użyć czegoś innego

Reply:
W tym przypadku nie było nic lepszego do wyboru. 
Zwróciłem na to uwagę :)